packages/react/src/themes/teams-dark/colors.ts

```diff
@@ -291,3 +291,40 @@ export const colorScheme: ColorSchemeMapping = {
     background: colors.yellow[100],
   }),
 }
+
+const createCategoryColorScheme = (color: string, customValues = {}) => {
+  return {
+    foreground1: categoryColors[color][250],
+    foreground2: categoryColors[color][550],
+    background: categoryColors[color][800],
+    ...customValues,
+  }
+}
+
+export const categoryColorSchemes: CategoryColorSchemeMapping = {
```

Comment:
Let's preserve the previous naming convention and rename this to `categoryColorScheme`